packages/fast-components-react-base/src/text-field/text-field.spec.tsx

```diff
@@ -94,4 +94,64 @@ describe("text-field", (): void => {
 
         expect(rendered.prop("placeholder")).toEqual("Test");
     });
+    /* tslint:disable no-console */
+    test("should throw a warning in the console if `TextFieldType.date` is passed to the type prop", () => {
+        console.warn = jest.fn();
```

Comment:
Is there a way of only decalring this once?

Reply:
moved to a new describe.

Reply:
Cool thanks